api/prisma/schema.prisma

```diff
@@ -23,4 +23,20 @@ model User {
   id       String @id @default(uuid())
   username String @unique
   password String
+
+  sessions Session[]
+}
+
+model Session {
+  id                  String    @id @default(uuid())
+  createdAt           DateTime  @default(now())
+  createdBy           String
+  referenceExpiryDate DateTime?
+  name                String?
+  revokedAt           DateTime?
+  referenceTokenId    String
+  isUserCreated       Boolean   @default(false)
+
+  creator User @relation(fields: [createdBy], references: [id], onDelete: Cascade)
+  @@unique([createdBy, id])
```

Comment:
Not sure why do you specify it like that, since you treat `session.id` as unique property. Let's make `id` primary/unique instead